cms/static/cms/js/modules/shortcuts/help.js

```diff
@@ -0,0 +1,53 @@
+var keyboard = require('keyboardjs');
+var Modal = require('../cms.modal');
+
+module.exports = function () {
+    var shortcutAreas = CMS.config.lang.shortcutAreas;
+    var modal = new Modal({
+        width: 600,
+        height: 600,
+        resizable: false,
+        minimizable: false,
+        maximizable: false
+    });
+
+    keyboard.setContext('cms');
+    keyboard.bind('?', function () {
+        modal.open({
+            title: CMS.config.lang.shortcuts,
+            width: 600,
+            height: 660,
+            html:
+                '<div class="cms-shortcuts">' +
```

Comment:
was originally a template string that i later transformed with babel :D
yeah, would make more sense i think